### ISSUE TITLE
Precompiled releases - tied to major version or minor of stemcell?

### ISSUE THREAD

**Issue by drnic** [cloudfoundry/bosh#1618]
In chatting with lovely Toronto bosh peeps, I came away with the idea that new compiled packages were reusable against any minor version of a common major version, say `3312.15` or `3312` or anything in between.

But when I ran `bosh2 export-release` it required an explicit `ubuntu-trusty/3363.9` minor version number; and independently when I unpack the bosh.yml compiled release and look in `release.MF` I see `stemcell: ubuntu-trusty/3312.15`:

```yaml
- name: ruby
  version: 589d4b05b422ac6c92ee7094fc2a402db1f2d731
  fingerprint: 589d4b05b422ac6c92ee7094fc2a402db1f2d731
  sha1: 04706fd16d8002385a1d69392a34fdea786e5bd2
  stemcell: ubuntu-trusty/3312.15
  dependencies: []
```

Do compiled releases (whilst they have a new incarnation inside normal release tgz) still tie to minor/exact stemcell versions? If so, is there a plan to make this more loose?

**Comment by cppforlife** [comment]
Added a note about floating: https://github.com/cloudfoundry/docs-bosh/commit/445d34a49b39586a2b93c56630e8b82a21b73fc0

**Comment by cppforlife** [comment]
Please reopen if doc update misses something.

**Comment by drnic** [comment]
Thanks; that reads well.






On Tue, Mar 14, 2017 at 8:55 PM -0400, "Dmitriy Kalinin" <notifications@github.com> wrote:












Closed #1618.
